staff/ismael-haytam/app-template/home.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Home</title>
+</head>
+<body>
+    <nav>
+        <ul>
+            <li><a href="./home.html">Home</a></li>
+            <li><a href="./news.html">News</a></li>
+            <li><a href="./login.html">Login</a></li>
+            <li><a href="./register.html">Register</a></li>
+            <li><a href="./about.html">About</a></li>
+            <li><a href="./profile.html">Profile</a></li>
+            <li><a href="./contact.html">Contact</a></li>
+        </ul>
+    </nav>
+    <header>
+        <h1>Quas doloribus sit rerum minus</h1>
+        <img src="https://picsum.photos/200/300" alt="random image">
+    </header>
+    <section>
```

Comment:
You can use "main" for englobe inside all "sections", nice code by the way.

Reply:
thanks ^^